point.py

```diff
@@ -0,0 +1,13 @@
+class Point(object):
+    def __init__(self,x,y,mark={}):
+        self.x = x
+        self.y = y
+        self.mark = mark
+
+    def check_coincident(self,point2):
+        check_this_point = (self.x,self.y)
+        return check_coincident(point1, point2)
+ 
+    def shift_point(self, x_shift, y_shift):
+        point = (self.x, self.y)

```

Comment:
This is a legitimate indentation issue.  Be careful as Python is able to avoid `{}` and `;` by enforcing strick indentation rules.